packages/ace-cli/src/index.js

```diff
@@ -45,6 +50,14 @@ function sleep(ms) {
 }
 
 async function run() {
+
+  if (cli.flags.rules) {
```

Comment:
this code should be called after the logger initialization, and use the logging API to output info.